pkg/daemon/daemon.go

```diff
@@ -51,17 +52,30 @@ type ProcessManager struct {
 
 // NewProcessManager is used by unit tests
 func NewProcessManager() *ProcessManager {
-	process := &ptpProcess{}
-	process.ptpClockThreshold = &ptpv1.PtpClockThreshold{
+	processPTP := &ptpProcess{}
+	processPTP.ptpClockThreshold = &ptpv1.PtpClockThreshold{
 		HoldOverTimeout:    5,
 		MaxOffsetThreshold: 100,
 		MinOffsetThreshold: -100,
 	}
 	return &ProcessManager{
-		process: []*ptpProcess{process},
+		process: []*ptpProcess{processPTP},
 	}
 }
 
+// SetTestProfile ...
+func (p *ProcessManager) SetTestProfileProcess(name string, ifaces config.IFaces, socketPath,
```

Comment:
can this we done by creating a test file with same package name ?

Reply:
worried that we are  creating more new changes in  older version , we should carry these tests from master